src/.bear-tidy-config

```diff
@@ -0,0 +1,15 @@
+{
+  "output": {
+    "content": {
+      "include_only_existing_source": true,
+      "paths_to_include": [],
+      "paths_to_exclude": [
+        "src/leveldb"
+      ]
+    },
+    "format": {
+      "command_as_array": true,
+      "drop_output_field": false
+    }
+  }
+}
```

Comment:
For reference: https://github.com/rizsotto/Bear/blob/master/source/citnames/man/citnames.1.md#config-file